docs/src/main/paradox/bootstrap/demos/index.md

```diff
@@ -0,0 +1,4 @@
+# Bootstrap examples 
```

Comment:
This page could be hard to find, because it's not listed at the top-level menu, only linked from the bottom of the bootstrap page.